rewrite-java/src/main/java/org/openrewrite/java/internal/grammar/AnnotationSignatureParser.java

```diff
@@ -412,6 +412,10 @@ public static class ElementValuePairContext extends ParserRuleContext {
 		public ElementValueContext elementValue() {
 			return getRuleContext(ElementValueContext.class,0);
 		}
+		public List<TerminalNode> SPACE() { return getTokens(AnnotationSignatureParser.SPACE); }
+		public TerminalNode SPACE(int i) {
+			return getToken(AnnotationSignatureParser.SPACE, i);
+		}
```

Comment:
I'm seeing "Cannot resolve symbol 'SPACE'" here; did anything go wrong with the generator?

Reply:
For context: typically we run the generator, and then follow up with `./gradlew licenseFormat` for the headers, and have recipes apply our best practices on top.